tests/php/Forms/FormTest.php

```diff
@@ -592,8 +592,8 @@ public function testDisableSecurityTokenDoesntAddTokenFormField()
             'Token field not added if disableSecurityToken() is set'
         );
     }
-
-    public function testDisableSecurityTokenAcceptsSubmissionWithoutToken()
+    
```

Comment:
```suggestion

```
The suggestion isn't very clear.... keep the new line, but remove the spaces.

Reply:
DONE